lib/matplotlib/backends/backend_qt.py

```diff
@@ -637,7 +637,7 @@ class NavigationToolbar2QT(NavigationToolbar2, QtWidgets.QToolBar):
         ("Customize", "Edit axis, curve and image parameters",
          "qt4_editor_options", "edit_parameters"))
 
-    def __init__(self, canvas, parent, coordinates=True):
```

Comment:
This doesn't seem to fall back to anything?

Reply:
That's intentional, qt widgets don't need to be parented initially (parent=None is just fine) and will be parented as needed when added to a parent window.